From 68fb3005c783a31fca33f77a7fc8bf53b94b8e35 Mon Sep 17 00:00:00 2001
From: Dan Douglass <dan.douglass@freescale.com>
Date: Wed, 20 Jan 2016 14:10:29 -0600
Subject: [PATCH 0901/5242] MLK-12303 Add CAAM support to the imx6QDL device
 tree for kernel v4.1

commit  d220f4e23f8c0779fbebce6b9db3197df52e1176 from
https://source.codeaurora.org/external/imx/linux-imx.git

The CAAM nodes have been added in imx6qdl.dtsi.

Signed-off-by: Dan Douglass <dan.douglass@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl.dtsi |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 6944266..6aac227 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -1027,7 +1027,8 @@
 				#address-cells = <1>;
 				#size-cells = <1>;
 				reg = <0x2100000 0x10000>;
-				ranges = <0 0x2100000 0x10000>;
+				ranges = <0 0x2100000 0x40000>;
+				interrupt-parent = <&intc>; /* interrupts = <0 92 0x4>; */
 				clocks = <&clks IMX6QDL_CLK_CAAM_MEM>,
 					 <&clks IMX6QDL_CLK_CAAM_ACLK>,
 					 <&clks IMX6QDL_CLK_CAAM_IPG>,
-- 
1.7.9.5

